linux_os/guide/system/software/sudo/sudo_add_passwd_timeout/rule.yml

```
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+prodtype: ol7,ol8,rhel7,rhel8
+
+title: 'Ensure sudo passwd_timeout is appropriate - sudo passwd_timeout'
```

I'd change the title to something nicer

```suggestion
title: 'Configure sudo password timeout'
```